# general
logging_level: 'INFO' # 10 #

test_option: ""

---
#Data info

dataset_root_dir: '/home/kowalski/data/MIPT_campus/winter-night'
sequence: '15-03-23-night'
segmentation_config: '/home/kowalski/data/MIPT_campus/anno_config.json'

semantics:
  initial_out_dir: 'init_semantics_conf'
  overwrite_existing_data: False
---
# Semantics 
semantic_uncert_range: 5.
semantic_refine_range: 15.

---

# Sensors
lidar:
  lower_bounds: [-50, -50, -50]
  upper_bounds: [50, 50, 50]

max_ts_delay: 0.05 # 5e-5



# q =  [w, x, y, z]
# t = [x, y, z]
front_cam:
  config_path: '/home/kowalski/data/MIPT_campus/calib/zed_calib.yml'
  left:
    lidar2cam_q: [-0.5098432024259881,-0.497602232228181, 0.49791462150659926, -0.4945027372133386]
    lidar2cam_t: [0.04632155672047202, -0.1327858803524132, -0.06092748914769486]

  right:
    lidar2cam_q: [-0.5098432024259881,-0.497602232228181, 0.49791462150659926, -0.4945027372133386] #! Not correct
    lidar2cam_t: [0.04632155672047202, -0.1327858803524132, -0.06092748914769486] #! Not correct
  
back_cam:
  config_path: '/home/kowalski/data/MIPT_campus/calib/realsense_calib.yml'
  left:
    lidar2cam_q: [-0.4996873443841852, -0.4927393159284719, -0.5006918959250938, 0.5067820706029872]
    lidar2cam_t: [0.01827032885440663, -0.14178753281358009, -0.0672336302622617]

  
# lidar2cam_T:
#   - [1, 0, 0, 0]
#   - [0, 1, 0, 0]
#   - [0, 0, 1, 0]
